common/txmgr/test_helpers.go

```diff
@@ -35,15 +35,17 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) XXXT
 }
 
 func (ec *Confirmer[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) XXXTestStartInternal() error {
-	return ec.startInternal()
+	ctx := context.Background()
```

Comment:
Can we get a context from the struct? Should avoid background as much as possible

Reply:
Yea, we can. Thanks!